src/Tracing/SpanContext.php

```diff
@@ -6,8 +6,6 @@
 
 class SpanContext
 {
-    private const TRACEPARENT_HEADER_REGEX = '/^[ \\t]*(?<trace_id>[0-9a-f]{32})?-?(?<span_id>[0-9a-f]{16})?-?(?<sampled>[01])?[ \\t]*$/i';
```

Comment:
This is a breaking change ⚠️ Instead of removing it, please add the following DocBlock to it

```php
/**
 * @deprecated since version 3.1, to be removed in 4.0
 */
```